tutorials/monitoring/4-endpoint-performance/index.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Monitoring NServiceBus solutions: Throughput and processing time"
+reviewed: 2017-10-10
+summary: Measuring individual endpoint performance with throughput and processing time.
+extensions:
+- !!tutorial
+  nextText: "Next Lesson: Queue length and critical time"
+  nextUrl: tutorials/monitoring/5-cross-endpoint-performance
+---
+
+include: monitoring-intro-paragraph
+
+This fourth lesson guides you through the throughput and processing time metrics and teaches you how to use those metrics to interpret the performance of an individual endpoint. Examples are shown using the sample monitoring solution.
+
+include: monitoring-sample-solution
+
+
+## Throughput and processing time
+
+Two of the simplest measures of endpoint performance are throughput and processing time. 
+
+Throughput is a measure of how much work the endpoint is doing. It is the rate at which the endpoint is able to process messages from it's input queue. While throughput can reflect endpoint performance, it is also heavily influenced by how much work there is to do. A highly optimized endpoint with only a few messages per second to process will still have a low throughput. 
+
+Processing time is a measure of the time it takes for the endpoint to process a single message. A higher processing time indicates a slower endpoint and a lower processing time indicates a faster endpoint. The amount of work assigned to an endpoint has less of an impact on processing time than it does on throughput. This makes it a better guage for individual endpoint performance.
```

Comment:
"Processing time is a measure of the time " -> "Processing time is the time ..."